scripts/client.js

```diff
@@ -0,0 +1,25 @@
+import KintoClient from "kinto-client";
+
+
+let client;
```

Comment:
We could also store client state in the store, and have a dedicated saga for acting on it. I'll spend some time playing around with these idea, thank you!